src/main/java/uk/ac/ebi/embl/api/validation/dao/EraproDAOUtilsImpl.java

```diff
@@ -392,10 +427,39 @@ public boolean isProjectValid(String project) throws SQLException
 		}
 	}
 
+	/**
+	 * Given SRA sample ID is used to look up corresponding Biosample accession which then is used to retreive sample
+	 * information from Biosamples first. If the sample could not be retrieved from Biosamples then it is loaded
+	 * from ERAPRO.
+	 *
+	 * @param sampleId SRA sample accession.
+	 * @return
+	 * @throws Exception
+	 */
 	@Override
 	public SourceFeature getSourceFeature(String sampleId) throws Exception {
-		SampleInfo sampleInfo = getSampleInfo(sampleId);
-		return new SourceFeatureUtils().constructSourceFeature(getSampleAttributes(sampleId), new TaxonomyClient(), sampleInfo);
+		SampleInfo sampleInfo = null;
```

Comment:
I recommend that we would just call BiosampleService in webin-cli-validator (eat your own dogfood principle):
```
new SampleService.Builder()
.setBioSampleUserName( Webin-40894 )
.setBioSamplePassword( <Webin superuser password stored in BioSamples> )
.setWebinUserName( su-Webin-N where Webin-N is the submittters submission account Id )
.setWebinPassword( <Webin superuser password stored in ERAPRO> )
.setTest( TEST )
.build();
...
```